internal/pkg/fixtures/fixtures.go

```diff
@@ -42,6 +43,25 @@ type Schedule struct {
 	Name string `json:"name"`
 }
 
+type Bucket struct {
+	ID          storageapi.BucketID `json:"id"`
+	Uri         string              `json:"uri"`
```

Comment:
`URI` ?

Reply:
omg, jasně

Reply:
Aha, ono je to tak už v go-clientovi (https://github.com/keboola/go-client/blob/f7f35faedc09266a8132e3c7b2fdf1917a9332ec/pkg/storageapi/bucket.go#L23). Ale opravím zatím aspoň tady.

Reply:
jj detail, v `go-client` sa to opravi, ked pojde niekto okolo a vsimne si to, ... ak by to nebolo rovno pod `ID`, tak by som si to tiez asi nevsimol.